sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -482,7 +526,7 @@ public void analyzeSentimentStringListWithOptions() {
             "The restaurant had amazing gnocchi."
         );
 
-        textAnalyticsAsyncClient.analyzeSentimentBatch(documents, "en", null).subscribe(
+        textAnalyticsAsyncClient.analyzeSentimentBatch(documents, "en", (TextAnalyticsRequestOptions) null).subscribe(
```

Comment:
Not a good example to show.
Please use the options more efficiently or use other overload. 

Applicable for all examples below.